app/assets/stylesheets/layouts/default.scss

```diff
@@ -0,0 +1 @@
+@import 'colorize'
```

Comment:
Missing semicolon?

Reply:
I think we don't _technically_ need one since that's the only line in the file, but also, yes.